modules/remotecontrol/Receiver.js

```diff
@@ -0,0 +1,191 @@
+/* global APP, JitsiMeetJS, interfaceConfig */
+const logger = require("jitsi-meet-logger").getLogger(__filename);
+import {DISCO_REMOTE_CONTROL_FEATURE, REMOTE_CONTROL_EVENT_TYPE, EVENT_TYPES,
+    PERMISSIONS_ACTIONS} from "../../service/remotecontrol/Constants";
+import RemoteControlParticipant from "./RemoteControlParticipant";
+import * as JitsiMeetConferenceEvents from '../../ConferenceEvents';
+
+const ConferenceEvents = JitsiMeetJS.events.conference;
+
+/**
+ * This class represents the receiver party for a remote controller session.
+ * It handles "remote-control-event" events and sends them to the
+ * API module. From there the events can be received from wrapper application
+ * and executed.
+ */
+export default class Receiver extends RemoteControlParticipant {
+    /**
+     * Creates new instance.
+     * @constructor
+     */
+    constructor() {
+        super();
+        this.controller = null;
+        this._remoteControlEventsListener
+            = this._onRemoteControlEvent.bind(this);
+        this._userLeftListener = this._onUserLeft.bind(this);
+        this._hangupListener = this._onHangup.bind(this);
+    }
+
+    /**
+     * Enables / Disables the remote control
+     * @param {boolean} enabled the new state.
+     */
+    enable(enabled) {
+        if(this.enabled !== enabled) {
```

Comment:
If this method is called twice with the same argument then it will execute twice. Is that ok ?

It can be at any point in future that something changes and make it execute twice (by mistake). I think we should detect that and either do not execute twice or log a warning.